app/views/home/index.haml

```diff
@@ -1,3 +1,5 @@
 .events.transitions-enabled
   = render @events
 = paginate @events
+#spinner
+  = icon('spinner', '', class: 'fa-pulse fa-4x')
```

Comment:
in my opinion spinner is to big and should have some more space between bottom of the screen :) but this is a matter of taste ;)